.eslintrc.json

```diff
@@ -0,0 +1,17 @@
+{
+  "rules": {
+    "require-atomic-updates": "off",
+    "no-unused-vars": "off",
+    "@typescript-eslint/no-unused-vars": ["error", {"vars": "all", "args": "none"}],
+    "no-redeclare": "off",
+    "@typescript-eslint/no-non-null-assertion": "error",
+    "@typescript-eslint/no-redeclare": "error",
+    "@typescript-eslint/no-explicit-any": "off",
+    "@typescript-eslint/no-var-requires": "off",
+    "@typescript-eslint/no-empty-function": "off"
+  },
+  "extends": ["plugin:@typescript-eslint/recommended", "prettier"],
```

Comment:
Q: Do we still want to extend `"plugin:fxa/recommended"` config and/or add the `ignorePatterns[]`?
Or are these all handled in the individual .eslintrc config files now?

Reply:
We want to remove `"plugin:fxa/recommended"` from our ecosystem entirely.

I should probably throw some ignorePatterns in here.